packages/sdk/src/oracle/services/statemachines/pollNewEvents.ts

```diff
@@ -55,6 +53,10 @@ export function Handlers(store: Store): GenericHandlers<Params, Memory> {
         memory.error = (err as unknown) as Error;
       }
 
+      // update our request table list with all known events
+      // this needs to be done outside the block sorted requests holds requests across different clients
+      await update.sortedRequests(chainId);
```

Comment:
this was a bug based on an edge case. the way the architecture was hacked to get this working was each oracle has its own client, but we have this `descendingRequests` array (a copy in each client) which contains all requests across all oracles and chains, which is a byproduct of this sorted request service. What was happening with this is that there were no requests on the goerli deployment so this update call never ran, and its local copy of `descendingRequests` was empty even though other chains had requests.  In the UI this may cause the main page to be empty if this client happened to be the default client chosen out of all of them to be checked for requests on the main index page. This call would populate the `descendingRequests` array. So i moved it out of the branch so its basically called every time we poll for new events.  This bug most likely would not show up in production, but I was seeing it in testing because of the unique circumstances.